docs/pywbemclicmdshelp.rst

```diff
@@ -686,23 +686,26 @@ The following defines the help output for the `pywbemcli connection add --help`
 
 ::
 
-    Usage: pywbemcli connection add [COMMAND-OPTIONS]
+    Usage: pywbemcli connection add [COMMAND-OPTIONS] NAME
 
       Add a new WBEM connection definition from specified options.
 
       Create a new WBEM connection definition in the connections file from the
```

Comment:
I suggest changing this sentence to be explicit that NAME is the new name, as follows: "Create a new WBEM connection definition named NAME in the connections file...".